config/vmdb.tmpl.yml

```diff
@@ -297,6 +297,7 @@ workers:
       :event_catcher_vmware:
         :poll: 1.seconds
         :ems_event_max_wait: 60
+        :ems_event_max_objects: 250
```

Comment:
@roliveri do you have a feel for how many inventory objects it took to start hitting the 2min timeout?